doc/source/_static/css/custom.css

```
@@ -486,3 +486,8 @@ footer {
 :root {
   --blue: #0475DE;
 }
+
+/* Make the hover color of tag/gallery buttons differ from "active" */
+.tag:hover {
```

Can we limit this lighter color to non-activated buttons? Otherwise when you're on a button it's impossible to see if it's currently activated or not
```suggestion
.tag.btn-outline-primary:hover {
```